test/e2e/util.go

```diff
@@ -242,7 +243,7 @@ func waitForPodRunningInNamespace(c *client.Client, podName string, namespace st
 			return true, nil
 		}
 		if pod.Status.Phase == api.PodFailed {
-			return true, fmt.Errorf("Giving up; pod went into failed status: \n%#v", pod.Status)
+			return true, fmt.Errorf("Giving up; pod went into failed status: \n%s", spew.Sprintf("%#v", pod))
```

Comment:
Fair enough, but I think in this case I actually want to slog through all that stuff, in case the diff obscures a detail I need.